treeherder/model/models.py

```diff
@@ -558,8 +558,7 @@ def update_after_verification(self, user):
             # classified this job.
             return
 
-        already_autoclassified = JobNote.objects.filter(failure_classification__name=classification, job=self).exists()
-        if already_autoclassified and user:
+        if user:
```

Comment:
At the only two callsites of `update_after_verification()` the user is always passed, having come from `request.user`. Do we need this `user` conditional at all?